src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -53,7 +53,7 @@ public class ReportBuilder {
     private static final ObjectMapper mapper = new ObjectMapper();
 
     private ReportResult reportResult;
-    private final ReportParser reportParser;
+    protected ReportParser reportParser;
```

Comment:
use getter/setter method instead and add unit test into https://github.com/damianszczepanik/cucumber-reporting/blob/5e084d9fc912910fea6d50c7939503aba57a21b4/src/test/java/net/masterthought/cucumber/ReportBuilderTest.java#L28

Reply:
Thank you for your reply @damianszczepanik! 
Let me try

Reply:
@damianszczepanik 
I have made the required changes. 
Please let me know if you have more suggestions.